scripts/splitControl.js

```diff
@@ -4,44 +4,32 @@
 /*
  * Create a splitter between two <div>s within a container.
  * Arguments for splitControl:
- * container - ID of a <div> which contains two <div>s (herein referred
+ * container - a <div> which contains two <div>s (herein referred
  * to as pane1 and pane2). The splitter will be created between these.
  * config - optional object that controls the div: (defaults in brackets)
  * {
  *   splitVertical: (true) true or false,
  *   splitPercent: (50), percentage of contaner occupied by pane1,
- *   reDraw - callback which should be fired when the container size changes.
- *       by default this is fired by window resize
- *       for a subsidiary splitter use onResize returned by the parent splitter
  *   dragBarSize: (6), the width/height of the splitterbar in pixels,
  *   dragBarColor: ("darkgray"),
```

Comment:
> Not relevant to this PR, but shouldn't styles and colors be set out in `styles/page_interfaces.less`

The split control is sort of generic rather than specifically pgdp and this is a default option for the drag bar colour. You could override it for each use case using data from  `styles/page_interfaces.less`. On the other hand it would be useful to be able to change it globally from our styles, perhaps if we give it a class also. I'll have a look at how this would work.